core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -35,7 +35,7 @@ import scala.collection.JavaConverters._
 import com.yammer.metrics.core.Gauge
 import org.apache.kafka.common.TopicPartition
 import org.apache.kafka.common.record.MemoryRecords
-import org.apache.kafka.common.requests.PartitionState
+import org.apache.kafka.common.requests.{PartitionState, PurgeResponse}
```

Comment:
Sorry, I thought checkstyle will catch all unused import but I am wrong. It seems that checkstyle only does this check for Java files. Will check it manually in the future.